python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -196,6 +196,12 @@ class PhotonTransferCurveExtractConfig(pipeBase.PipelineTaskConfig,
         doc="Minimum number of good data values to compute KS test histogram.",
         default=100,
     )
+    ksHistSampleMultiplier = pexConfig.Field(
+        dtype=int,
+        doc="Multiplier for how many more points to sample from the model to reduce measurement "
+            "noise.",
+        default=10,
+    )
```

Comment:
(Comment I forgot on the first pass)
Is there a reason to sample the model for this test instead of using the known Gaussian CDF with `ks_1samp`?  I doubt it matters, but this multiplier made me wonder if just using the CDF would be easier.

Reply:
That is a great point!  I have refactored to use the `ks_1samp` and gaussian cdf.